webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const SRC_DIR = path.join(__dirname, '/client/src');
+const DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  },
+  module : {
+    rules : [
+      {
+        test : /\.jsx?/,
+        include : SRC_DIR,
+        loaders : 'babel-loader',
+        // In webpack 4.x.x. presets needs to be defined in babelrc
+        // options: {
+        //   presets: ['react', 'env']
+        // }
+      }
+    ]
+  }
+};
+
+
+// module.exports = {
```

Comment:
@tegsheeb Do you still need this portion? (Line 27 Onwards)

Reply:
Thanks for pointing it out. No, we don't need it. Just cleaned up now.